### ISSUE TITLE
Security components: SummaryCardContainer a11y issue

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#1786]
## What package(s) are you using?

- [ ] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [ ] Legacy/CDAI
- [X] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.

After the latest dependency update, one of the security components now has an accessibility violation that causes several tests to fail.

<img width="1179" alt="image" src="https://user-images.githubusercontent.com/10215203/158443789-f4b1099c-0549-4b83-9191-7803b56a3d9e.png">

> Is this issue related to a specific component?

Security `SummaryCardContainer`
> What did you expect to happen? What happened instead? What would you like to
> see changed?

The security tests to pass without any errors
> What browser are you working in?

All
> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?

Latest
> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

N/A
## Steps to reproduce the issue

1. Remove the test.skip from the SummaryCard component tests and you'll be able to reproduce the failing tests.

## Additional information

- Screenshots or code
- Notes

**Comment by elycheea** [comment]
Going to keep as https://github.com/carbon-design-system/ibm-products/labels/status%3A%20wontfix since `SummaryCard` is not marked as legacy/deprecated.
